### PR TITLE
Added round function

### PR DESCRIPTION
**Pull request by Ritu01-git**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [ ] I've read the contribution guidelines.
- [ ] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: C/math/stdout.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+DOSBox version 0.74
```
Comment (iamrajiv) [inline C/math/stdout.txt:1]: @Ritu01-git Remove this file.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

File: C/math/stdout.txt (hunk 1 above)
Comment (Ritu01-git) [reply C/math/stdout.txt:1]: @iamrajiv removed  this file.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@Ritu01-git Remove `stdout.txt` and `stdout.txt` files they are not relevant.

**Review by atarax665 — CHANGES_REQUESTED** [review]
Make your forked repo even with the main repo. You can use [this](https://stackoverflow.com/questions/7244321/how-do-i-update-a-github-forked-repository) for reference.

**Comment by HarshCasper** [comment]
@Ritu01-git You don't seem to have worked on an Issue. I am closing the PR in the case because we don't accept PRs with no Issues attached to them. 

Make an Issue and reopen the PR by joining the [Slack](https://join.slack.com/t/tesseractcodi-bxq5968/shared_invite/zt-ju6c0nqq-KgkTvFyxjNVbDRBIjADanw) and dropping me a message.

**Pull request closed without merging**